source/runner/runners/utils/transpiler.ts

```diff
@@ -126,7 +126,7 @@ export const typescriptify = (content: string, dir: string): string => {
   let compilerOptions: any
   const tsConfigPath = lookupTSConfig(dir)
   if (tsConfigPath) {
-    compilerOptions = JSON5.parse(fs.readFileSync(tsConfigPath, "utf8"))
+    compilerOptions = parseTsconfig(tsConfigPath)
```

Comment:
This project already depends on TypeScript (see line `123`), so it’s not helpful for us to pull in a 3rd-party package whose purpose is to `parseTsconfig` without a dependency on TypeScript!

Instead, I’d accept a PR that uses`ts.parseConfig` or a similar API that supports `extends` from the official dependency — that we’re already pulling in.